src/components/GordonDialogBox/index.tsx

```diff
@@ -37,7 +37,7 @@ const GordonDialogBox = ({
       event.key === 'Enter' &&
       event.currentTarget.classList.contains('MuiDialog-root')
     ) {
-      buttonClicked(event);
+      buttonClicked!(event);
```

Comment:
A better approach would be to only call the function if the `buttonClicked` prop is not undefined:
```suggestion
      buttonClicked &&
      event.key === 'Enter' &&
      event.currentTarget.classList.contains('MuiDialog-root')
    ) {
      buttonClicked(event);
```